src/additional-tachyons/_animation.css

```diff
@@ -8,3 +11,44 @@
 @media(--breakpoint-large) {
   .ta-l { transition: all 0.3s; }
 }
+
+/*****************
+ * Spin
+ ****************/
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
```

Comment:
This is either a nit or a misunderstanding, but since 360ᵒ == 0ᵒ, wouldn't that result in a bit of a stutter in the transition from 100% to zero%? I'm visualizing this like clock hands, which may not be the correct visualization device.